R/causal_impact.R

```diff
@@ -34,6 +34,13 @@ do_causal_impact <- function(df, time, formula, ...) {
 #' @param time_col - Column that has time data
 #' @param formula - Formula with target value column on the left-hand side, and predictor columns on the right-hand side. e.g. y ~ predictor1 + predictor2
 #' @param intervention_time - The point of time where intervention happened.
+#' @param na_fill_type - Type of NA fill:
```

Comment:
How about making this argument na_fill and if it is numeric, regarding that it's the value to fill NA?